ui/src/app/applications/components/application-operation-state/application-operation-state.tsx

```diff
@@ -93,7 +93,15 @@ export const ApplicationOperationState: React.StatelessComponent<Props> = ({appl
         });
     }
     if (operationState.syncResult) {
-        operationAttributes.push({title: 'REVISION', value: <Revision repoUrl={utils.getAppDefaultSource(application).repoURL} revision={operationState.syncResult.revision} />});
```

Comment:
Previously, the revision value wouldn't be displayed until the operation has finished.  Seems like in this new behavior, the revision represents the most recent, successfully synced revision, and therefore will always show here, even if the operation is in progress.  Is this the intended design?  Is the if check on syncResult needed?